README.md

```diff
@@ -0,0 +1,44 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `yarn start`
```

Comment:
Why yarn here and npm in the backend?

Reply:
Ah crap no i'm using npm here, but the readme is wrong. I would personally prefer yarn but probably easier if we use as little extra tools as possible. Not large projects either.